Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -750,4 +750,24 @@ object LocalData {
      ****************************************************/
 
     fun getSharedPreferenceInstance(): SharedPreferences = globalEncryptedSharedPreferencesInstance
+
+    /****************************************************
+     * INTEROPERABILITY
+     ****************************************************/
+
+    fun saveInteroperabilityUsed() {
+        getSharedPreferenceInstance().edit().putBoolean(
+            CoronaWarnApplication.getAppContext()
+                .getString(R.string.preference_interoperability_is_used_at_least_once),
```

Comment:
Don't use `R.string.` for shared preference keys, downsides are huge (localization impacting data storage), while upsides are small. Add the key as `const val` to the class.

Reply:
Would also be my prefered way, but all other methods were done this way so I just kept with it. Would be great to refactor all of them at once

Reply:
> Would also be my prefered way, but all other methods were done this way so I just kept with it. Would be great to refactor all of them at once.

That's okay, one step at a time, please use the better approach here already, need to start somewhere :grin:.

Reply:
Will do it 👍